belgif-rest-problem-java-ee/src/test/java/io/github/belgif/rest/problem/jaxrs/client/ProblemClientResponseFilterTest.java

```diff
@@ -97,4 +98,13 @@ void differentMediaType() {
                 () -> filter.filter(requestContext, responseContext));
     }
 
+    @Test
+    void microProfile() {
+        when(requestContext.getProperty("org.eclipse.microprofile.rest.client.invokedMethod"))
```

Comment:
Is there a way to test that in the integration test ? My fear is that when the property value changes, you will go back to the old behaviour.

Reply:
The MicroProfile REST Client spec defines that this property MUST be set: https://download.eclipse.org/microprofile/microprofile-rest-client-4.0/microprofile-rest-client-spec-4.0.html#_clientrequestfilter.

I did verify it manually (looking at the code coverage for the integration test).